biz.aQute.bnd.gradle/README.md

```diff
@@ -307,12 +307,13 @@ in the file. For example:
 ```groovy
 import aQute.bnd.gradle.Resolve
 
-tasks.register('resolve', Resolve) {
+def resolveTask = tasks.register('resolve', Resolve) {
   bndrun 'my.bndrun'
+  outputBndrun layout.buildDirectory.file('my.bndrun')
 }
 ```
 
-There are six properties which can be configured for a Resolve task:
+There are eight properties which can be configured for a Resolve task:
```

Comment:
To save you some typing in the future you could change this to:
> Following are the properties which can be configured for a Resolve task:

Just a suggestion.